python/lsst/ip/isr/isrMockLSST.py

```diff
@@ -469,13 +570,31 @@ def makeImage(self):
 
             # 3. Add BF effect (electron) to imaging portion of the amp.
             if self.config.doAddBrighterFatter is True:
-                self.amplifierAddBrighterFatter(ampImageData,
-                                                rngBrighterFatter,
-                                                self.config.bfStrength,
-                                                self.config.nRecalc)
+                self.amplifierAddBrighterFatter(
+                    ampImageData,
+                    rngBrighterFatter,
+                    self.config.bfStrength,
+                    self.config.nRecalc,
+                )
 
             # 4. Add serial CTI (electron) to amplifier (imaging + overscan).
-            # TODO
+            if self.config.doAddDeferredCharge:
+                # Get the free charge area for the amplifier.
+                bboxFreeCharge = amp.getRawDataBBox()
+                bboxFreeCharge = bboxFreeCharge.expandedTo(amp.getRawHorizontalOverscanBBox())
+                bboxFreeCharge = bboxFreeCharge.expandedTo(amp.getRawHorizontalPrescanBBox())
+                bboxFreeCharge = bboxFreeCharge.expandedTo(amp.getRawVerticalOverscanBBox())
+                ampFreeChargeData = exposure.image[bboxFreeCharge]
+
+                self.amplifierAddDeferredCharge(
+                    amp,
+                    ampImageData,
+                    ampFreeChargeData,
+                    cti=self.deferredChargeCalib.globalCti[amp.getName()],
+                    traps=[self.deferredChargeCalib.serialTraps[amp.getName()]],
+                    driftScale=self.deferredChargeCalib.driftScale[amp.getName()],
+                    decayTime=self.deferredChargeCalib.decayTime[amp.getName()],
+                )
```

Comment:
I think that this function should do more of this.  Basically, if the function call was:
`self.amplifierAddDeferredCharge(exposure, amp, self.deferredChargeCalib)` then the main function would be easier to read, and I think the function itself would have everything that it needs to extract `ampImageData`, `ampFreeChargeData`, and all of the cti/traps/driftScale/decayTime.  And you would be able to simplify the docstring below by a lot!  😄